.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# CODEOWNERS
+#
+# documentation for this file can be found at:
+# https://help.github.com/en/articles/about-code-owners
+
+# These are the default owners for the code and will
+# be requested for review when someone opens a pull request.
+# Order is alphabetical for easier maintenance.
+#
+# All Terraform Engineers in Dell (dell/TerraformDell)
+
+# for all files:
```

Comment:
Is it the correct DL?

Reply:
@gokul-srivathsan It intents to use github team as the code owners. @org/teamname is the syntax of referenecing a team in codeowners file.

Reply:
Weird. Github shows
> Unknown owner on line 13: make sure the team @dell/TerraformDell exists, is publicly visible, and has write access to the repository
* @dell/TerraformDell

Reply:
Add write access to terraformdell team that removes the warning.